app/src/main/res/layout/videoplayer_activity.xml

```diff
@@ -124,4 +124,20 @@
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/seek_display"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="5dp"
+        android:layout_gravity="center"
+        android:background="#c0000000"
+        android:paddingLeft="30dp"
+        android:paddingTop="5dp"
+        android:paddingRight="30dp"
+        android:paddingBottom="5dp"
```

Comment:
Please replace with multiples of 4 (or 8) to comply with the Material Design guidelines.

Reply:
No problem I replaced the 5's with 4's and the 30's with 24's here is the result.
![Screenshot_1582845779](https://user-images.githubusercontent.com/49007921/75496325-1158b900-597e-11ea-9a45-7e7159b5ff86.png)

Reply:
> @dallinpstock: No problem I replaced the 5's with 4's and the 30's with 24's here is the result.

I am happy to merge the PR as-is if you could push the changes